### PR TITLE
[Auth] use logged_out_user for auth

### PR DESCRIPTION
**Pull request by l2succes**
depends on: https://github.com/artsy/reaction/pull/723

### CODE REVIEW

File: src/desktop/apps/auth2/client/index.tsx [hunk 1]
```diff
@@ -51,7 +54,27 @@ export const initModalManager = () => {
...
         csrf={sd.CSRF_TOKEN}
-        redirectUrl="/"
+        handleSubmit={(type, values) => {
+          user.set(values)
+          const options = {
+            success: () => {
+              document.location.hash = '/'
+            },
+            error: (_, response) => {
+              console.log(response)
```
Comment (kanaabe) [inline src/desktop/apps/auth2/client/index.tsx]: Stray log

File: src/desktop/apps/auth2/client/index.tsx [hunk 2]
```diff
@@ -51,7 +54,27 @@ export const initModalManager = () => {
...
+              mediator.trigger('auth:error', message)
+            },
+          }
+          switch (type) {
+            case ModalType.login:
+              user.login(options)
+              break
+            case ModalType.signup:
+              user.signup(options)
+              break
```
Comment (kanaabe) [inline src/desktop/apps/auth2/client/index.tsx:75]: Whew! Nice that this worked out and ended up being less overhead for us.

**Comment by kanaabe** [comment]
LGTM! You can merge since I don't have permission

File: src/desktop/apps/auth2/client/index.tsx (hunk 1 above)
Comment (l2succes) [reply src/desktop/apps/auth2/client/index.tsx]: 👍

**Pull request merged**